app/src/main/java/hm/orz/chaos114/android/slideviewer/ui/SelectOcrLanguageActivity.java

```diff
@@ -83,7 +85,7 @@ protected void onCreate(Bundle savedInstanceState) {
         binding.languageList.setAdapter(adapter);
     }
 
-    private class Adapter extends BaseAdapter {
+    private class Adapter extends BaseAdapter { 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/SlideViewer/blob/eeecb317459f4cd83100228fe8ffde0ce9f030fb/app/src/main/java/hm/orz/chaos114/android/slideviewer/ui/SelectOcrLanguageActivity.java#L88"></span>Class Adapter should be declared as final.</td>
    </tr>
  </tbody>
</table>